src/NodaTime.Web/Markdown/unstable/core-types.md

```diff
@@ -0,0 +1,143 @@
+@Title="Core types quick reference"
```

Comment:
Sigh, git doesn't record where copies come from, so I had to clone the patch locally and run diff with `--find-copies-harder` to get a good diff.